crates/sncast/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "cast"
```

Comment:
[[bin]] section at the bottom could be removed now

Reply:
I don't see much benefit in that. I know that there are some default values for `[[bin]]` target, but we don't have to rely on them.